### DISCUSSION TITLE
How do I change i18n build dir?

### DISCUSSION THREAD

**Discussion by zhengyimeng** [nuxt-modules/i18n#2102] [Q&A]
### Environment

npx build

### Reproduction

无

### Describe the bug

i18n 构建之后的默认目录是 .output/public/__i18n__
我如何修改 __i18n__ 为其他的名称？
能否增加一个选项叫，buildAssetsDir 改变 i18n 的默认路径？

**Comment by kazupon** [comment]
This is not issue, question, so we will convert to github discussions

**Reply by kazupon** [reply]
Sorry, I'm not sure what exactly **i18n** in your question refers to.

If you mean the message resources placed in the `lang` option of the nuxt i18n module, then unfortunately we cannot specify the output directory at the moment.

As FYI, You can see the configuration about directory.
https://nuxt.com/docs/api/configuration/nuxt-config#dir
